Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/MainActivity.kt

```diff
@@ -112,6 +114,7 @@ class MainActivity : AppCompatActivity() {
                 resources.getString(R.string.str_social) -> fragment = SocialFragment()
                 resources.getString(R.string.str_coc) -> fragment = CocFragment()
                 resources.getString(R.string.str_about_us) -> fragment = AboutFragment()
+                resources.getString(R.string.str_speakers) -> fragment = SpeakerFragment()
             }
             // Add fragment with tag
             supportFragmentManager.beginTransaction().add(R.id.fragment_container, fragment, title).commit()
```

Comment:
We can use replace in the following code to avoid fragment overlap issues 🤔 

`supportFragmentManager.beginTransaction().add(R.id.fragment_container, fragment, title).commit()`

Reply:
`replace` did fix the issue. 

Was it not the expected case that the call to `supportFragmentManager.findFragmentByTag` would have been a non-null fragment  and the `detach`/`attach` is called instead?